tests/release/README.md

```diff
@@ -0,0 +1,46 @@
+# Release e2e tests
+
+Contains E2E tests related to [Release Service](https://github.com/redhat-appstudio/infra-deployments/tree/main/components/release).
+
+In this document will be detailed the included e2e tests for release suitre.
+
+## e2e tests
+### 1. Good Path (release.go)
+   This test verifies the functionality of the release-service in detecting snapshots and performing the necessary actions based on the design.
+
+      Steps:
+       - Create the following CRDs: Snapshot, Release Strategy, Release Plan, Enterprise Contract, ReleasePlanAdmission, and Release.
+
+      Expected Results:
+      - The release PipelineRun is successfully created in the managed namespace.
+      - The PipelineRun should pass.
+      - Validate that the Release passes.
+      - Validate that the Release object is referenced to the PipelineRun.
+
+### 2. Good Path (e2e-test-default-bundle.go)
+   Test focuses on running the release pipeline when a user creates all the required CRDs. Once the component is created, we expect the build pipeline to pass, followed by the creation of a snapshot with the component information. This triggers the creation of a release object, and both the release pipeline and release object are expected to pass.
```

Comment:
The snapshot being created with the component information really has nothing at all to do with the release service